command/genesis/genesis.go

```diff
@@ -148,6 +146,34 @@ func setFlags(cmd *cobra.Command) {
 			"the maximum number of validators in the validator set for PoS",
 		)
 	}
+
+	// PolyBFT
+	{
+		cmd.Flags().IntVar(
+			&params.validatorSetSize,
+			validatorSetSizeFlag,
+			defaultValidatorSetSize,
+			"validator set size",
+		)
+		cmd.Flags().Uint64Var(
+			&params.sprintSize,
+			sprintSizeFlag,
+			defaultSprintSize,
+			"sprint size",
+		)
+		cmd.Flags().DurationVar(
+			&params.blockTime,
+			blockTimeFlag,
+			defaultBlockTime,
+			"block time",
+		)
+		cmd.Flags().StringArrayVar(
+			&params.validators,
+			validatorsFlag,
+			[]string{},
+			"validators list (format: <address>:<blskey>)",
```

Comment:
Please add a better description for this flag 🙏

Reply:
fixed